privacyidea/lib/smsprovider/HttpSMSProvider.py

```diff
@@ -291,10 +277,7 @@ def parameters(cls):
                           "values": ["yes", "no"]
                       },
                       "REGEXP": {
-                          "description": _("Regular expression to modify the phone number "                 
-                                           "to make it compatible with provider. "
-                                           "Enter something like '/[\\+/]//' to remove "
-                                           "pluses and slashes.")
+                          "description": cls.REGEXP_description
```

Comment:
We should use lower case parameter names. The upper case ones are from a time where the configuration for the providers was kept separately.
We could keep it uppercase here (and lower case in the ScriptSmsProvider) and convert it to lower case when using it in `_mangle_phone()`